nixos/modules/services/networking/iodine.nix

```diff
@@ -130,7 +130,12 @@ in
         createIodineClientService = name: cfg:
           {
             description = "iodine client - ${name}";
-            after = [ "network.target" ];
+
+            # Seems to save bad network state, breaking it if it is started too
+            # early relative to the network.
```

Comment:
Was this just always flaky? Or is this breakage new?

Reply:
I think this breakage might be new, but I absolutely don't understand why it is happening, given that it is resolved by depending on network-online.target.